zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Paragraph.java

```diff
@@ -61,6 +61,7 @@
   Date dateUpdated;
   private Map<String, Object> config; // paragraph configs like isOpen, colWidth, etc
   public final GUI settings;          // form and parameter settings
+  private Map<String, ParagraphRuntimeInfos> runtimeInfos;
```

Comment:
Should we rename `ParagraphRuntimeInfos` to `ParagraphRuntimeInfo` ? Because it is a little confusing to see `runtimeInfos` also plural